test/Typewriter.Test/TestDataPHP/com/Microsoft/Graph/CallRecords/Model/CallRecord.php

```diff
@@ -58,8 +58,8 @@ public function setVersion($val)
     */
     public function getType()
     {
-        if (array_key_exists("type", $this->_propDict)) {
-            if (is_a($this->_propDict["type"], "\Microsoft\Graph\CallRecords\Model\CallType") || is_null($this->_propDict["type"])) {
+        if (array_key_exists("type", $this->_propDict) && !is_null($this->_propDict["type"])) {
+            if (is_a($this->_propDict["type"], "\Microsoft\Graph\CallRecords\Model\CallType")) {
```

Comment:
What is the reason to perform this type check? If it isn't the expected type, it seems we have another issue and shouldn't be providing a new empty instance of this type.

Reply:
Yes, we have an issue since we don't have type constraints in the setters.

Reply:
We will be working on a fix once we get to start work on Kiota.

Reply:
Will be addressed with https://github.com/microsoftgraph/msgraph-sdk-php/issues/530

Reply:
Yep, this was already here. Resolving